lib/spack/spack/cmd/module.py

```diff
@@ -257,6 +257,14 @@ def module(parser, args):
     }
     query_args = constraint_qualifiers.get(args.subparser_name, {})
     specs = args.specs(**query_args)
+    if len(specs) > 1:
```

Comment:
@healther This is in the `module` command. I didn't try it, but it seems it will prevent the regeneration of multiple modules at once. What happens if you try:
```
$ spack module refresh -m tcl
```
using this branch?